addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -155,3 +155,56 @@ def test_mailing_list_merge_cornercase(self):
         merge = merge_form.save()
         self.assertEqual(merge.src_list_ids, self.mailing_list_1 + self.mailing_list_2)
         self.assertEqual(merge.dest_list_id, self.mailing_list_3)
+
+
+class TestMailingContactImport(MassMailCommon):
+    """Test the wizard <mailing.contact.import>."""
+
+    @users('user_marketing')
+    def test_mailing_contact_import(self):
+        mailing_list, other_list = self.env['mailing.list'].create([
+            {'name': 'Test mailing list'},
+            {'name': 'Other mailing list'},
+        ])
+
+        self.env['mailing.contact'].create([
+            {
+                'email': 'already_exists@example.com',
+                'list_ids': mailing_list.ids,
+            }, {
+                'email': 'already_exists_in_other_list@example.com',
+                'list_ids': other_list.ids,
+            }
+        ])
+
+        contact_import = self.env['mailing.contact.import'].create({
```

Comment:
Would use the form tool just to have complete tests involving the view itself, ...